app/ante.go

```diff
@@ -12,13 +12,19 @@ import (
 	errorsmod "cosmossdk.io/errors"
 	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
 	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	// TODO: revisit
```

Comment:
Should remove those unused modules first, you can create a PR to add that later

Reply:
sure, it's removed for now